css/base/base.css

```diff
@@ -13,6 +13,12 @@
 
   --grey: hsl(207, 21%, 48%);
   --dark-grey: hsl(207, 25%, 28%);
+
+  --alert-primary: 	hsl(239, 84%, 67%);
+  --alert-secondary: hsl(0, 8%, 60%);
+  --alert-success: hsl(153, 53%, 53%);
```

Comment:
can rename to success because generally over the whole website, it will have the same color.

Reply:
done